pros/examples/accessories.rs

```diff
@@ -6,10 +6,21 @@ use pros::prelude::*;
 
 #[derive(Debug, Default)]
 struct ExampleRobot;
-impl Robot for ExampleRobot {
-    fn opcontrol(&mut self) -> pros::Result {
+#[async_trait]
+impl AsyncRobot for ExampleRobot {
+    async fn opcontrol(&mut self) -> pros::Result {
+        let handle = pros::async_runtime::spawn(async {
+            for _ in 0..5 {
+                println!("Hello from async!");
+                sleep(Duration::from_millis(1000));
```

Comment:
Should this be awaited?

Reply:
fixed along with other mistakes in the same file.